app/src/main/res/values/strings.xml

```diff
@@ -35,6 +35,13 @@
     <string name="complete_name">Full Name</string>
     <string name="delete_account">Delete Account</string>
 
+    <string name="list_button_text">List</string>
```

Comment:
Please, add a small commentary to tag this section as the offline mode components